api/metrics-api-jaxrs/pom.xml

```diff
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hawkular-metrics-parent</artifactId>
     <groupId>org.hawkular.metrics</groupId>
-    <version>0.3.0-SNAPSHOT</version>
+    <version>0.3.1-SNAPSHOT</version>
```

Comment:
We will probably not release 0.3.0, with snapshots getting published on every build there is no incentive to make this a public official release. 

Once everybody catches up on 0.3.1 or newer we should make the first official release of Hawkular Metrics.